packages/node/src/lib/metrics-log.ts

```diff
@@ -96,7 +98,7 @@ export function metricsAPICall(
   const makeRequest = () => {
     if (backoffExpiresAt) {
       if (backoffExpiresAt > new Date()) {
-        return Promise.resolve();
+        return Promise.resolve(undefined);
```

Comment:
```suggestion
        return Promise.resolve(null);
```

While it definitely maps closer to the existing behavior, returning `Promise<undefined>` just seems... wrong.

Reply:
resolving null doesn't typecheck, unfortunately:

```
Diagnostics:
Type 'Promise<LogResponse | { response: Response | null; ids: LogId; }>' is not assignable to type 'Promise<LogResponse>'.
  Type 'LogResponse | { response: Response | null; ids: LogId; }' is not assignable to type 'LogResponse'.
    Type '{ response: Response | null; ids: LogId; }' is not assignable to type 'LogResponse'.
      Types of property 'response' are incompatible.
        Type 'Response | null' is not assignable to type 'Response | undefined'.
          Type 'null' is not assignable to type 'Response | undefined'. [2322]
```

because LogResponse's `response` field is optional, it must be either `Response` or `undefined`

Reply:
yeah that's a bigger logic change, something I'd like to keep out of this PR if possible!